homeassistant/components/nordpool/__init__.py

```diff
@@ -0,0 +1,29 @@
+"""The Nord Pool component."""
+
+from __future__ import annotations
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.util import dt as dt_util
+
+from .const import PLATFORMS
+from .coordinator import NordpooolDataUpdateCoordinator
+
+type NordpoolConfigEntry = ConfigEntry[NordpooolDataUpdateCoordinator]
```

Comment:
Is it Nord Pool or NordPool or Nordpool? Because if its either of the first 2, the camel casing is off